packages/tools/babylonServer/public/recast.d.ts

```diff
@@ -81,6 +81,7 @@ declare module "recast-detour" {
     moveAlong(position: Vec3, destination: Vec3): Vec3;
     getNavMesh(): dtNavMesh;
     computePath(start: Vec3, end: Vec3): NavPath;
+    computePathSmooth(start: Vec3, end: Vec3): NavPath;
```

Comment:
Have you released a new npm package as well?

Reply:
nope, not yet.

Reply:
release is triggered by master changes

Reply:
Should happen in parallel with merging this PR. Not too urgent, as the next minor version is in 5 days